modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliOptions.cs

```diff
@@ -149,17 +147,17 @@ public class LegacyCliOptions : Dictionary<string, string>, IAgentConfigProvider
                         (int i) => this[LegacyCliConfigKeys.IoTHubMaxMessageSize] = i.ToString() },
                     { "om|maxoutgressmessages=", "The maximum size of the (IoT D2C) message outgress buffer",
                         (int i) => this[LegacyCliConfigKeys.MaxOutgressMessages] = i.ToString() },
-
-                    // testing purposes
-                    { "sc|scaletestcount=", "The number of monitored item clones in scale tests.",
-                        (int i) => this[LegacyCliConfigKeys.ScaleTestCount] = i.ToString() },
                     { "mm|messagingmode=", "The messaging mode for messages " +
                         $"(allowed values: {string.Join(", ", Enum.GetNames(typeof(MessagingMode)))}).",
                         (MessagingMode m) => this[LegacyCliConfigKeys.MessagingMode] = m.ToString() },
                     { "me|messageencoding=", "The message encoding for messages " +
                         $"(allowed values: {string.Join(", ", Enum.GetNames(typeof(MessageEncoding)))}).",
                         (MessageEncoding m) => this[LegacyCliConfigKeys.MessageEncoding] = m.ToString() },
 
+                    // testing purposes
+                    { "sc|scaletestcount=", "The number of monitored item clones in scale tests.",
```

Comment:
Should we also have a dedicated command-line argument for maximum nodes per dataset ?

Reply:
Not really. I added it in the first place and then removed it. Let's keep this an an environment variable in the same manner we handle the "newly" added configuration defaults like "MaxOutgressMessages". I think we should slowly start using these format of arguments and slowly deprecate the 2.5.x specific format. These are only available in the standalone (legacy) monde anyhow.

Reply:
I am also thinking about losing mention of it in the help. But that is a minor thing, I guess.